src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbs.tsx

```diff
@@ -210,54 +228,85 @@ class Breadcrumbs extends React.Component<Props, State> {
     };
   };
 
-  getCollapsedCrumbQuantity = (): {
-    filteredCollapsedBreadcrumbs: BreadcrumbsWithDetails;
-    collapsedQuantity: number;
-  } => {
-    const {isCollapsed, filteredBreadcrumbs} = this.state;
-
-    let filteredCollapsedBreadcrumbs = filteredBreadcrumbs;
-
-    if (isCollapsed && filteredCollapsedBreadcrumbs.length > MAX_CRUMBS_WHEN_COLLAPSED) {
-      filteredCollapsedBreadcrumbs = filteredCollapsedBreadcrumbs.slice(
-        -MAX_CRUMBS_WHEN_COLLAPSED
-      );
-    }
-
-    return {
-      filteredCollapsedBreadcrumbs,
-      collapsedQuantity: filteredBreadcrumbs.length - filteredCollapsedBreadcrumbs.length,
-    };
+  scrollToTheBottom = (element: HTMLDivElement) => {
+    element.scrollTo({
+      top: element.scrollHeight,
+      left: 0,
+      behavior: 'smooth',
+    });
```

Comment:
```suggestion
    element.scrollTo({
      top: element.scrollHeight,
      left: 0,
    });
```
I would remove the scrolling animation since it doesn't serve a real purpose.
I tried it locally and for me it feels better without the animation.